specification/storage/resource-manager/Microsoft.Storage/preview/2023-04-01-preview/blob.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "2023-01-01",
+    "version": "2023-04-01",
```

Comment:
Will the API version be "2023-04-01" or "2023-04-01-preview"?
The API version in the file path and in swagger should be aligned.